### PR TITLE
Create pop-up

### PR DESCRIPTION
**Pull request by JoyTaribagshaw**
#Highlights 

- [ ] Update JS file 
- [ ] Created Popup for both mobile and desktop screens
- [ ] Ensured all linters pass
- [ ] Below is an image of the pop on both desktop and mobile

# Mobile Screenshot
mobile-popup

# Desktop Screenshot

desktop-popup

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+document.getElementById('seeProjectButton').addEventListener('click', (event) => {
+  event.preventDefault();
+  // const button = event.target;
+  const mobilePopup = document.getElementById('mobile-popup');
+  // const buttonRect = button.getBoundingClientRect();
+  // const buttonCenterX = buttonRect.left + buttonRect.width + 25;
+  // mobilePopup.style.left = `${buttonCenterX}px`;
+  // mobilePopup.style.top = `${buttonRect.top - mobilePopup.offsetHeight}px`;
```
Comment (Alusp) [inline index.js]: - [ ] You have done great job :clap: However, Please do not commit old pieces of code as inline comments. Leaving old pieces of code as comments can clutter the codebase and make it harder to read and understand. Over time, as more code comments accumulate, it becomes challenging to differentiate between actual code and commented-out sections. This clutter can hinder the development process and introduce confusion for developers trying to navigate and maintain the code. :slightly_smiling_face:

File: style.css [hunk 2]
```diff
@@ -553,10 +594,121 @@ textarea {
...
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  margin: 20px 3px;
+  cursor: pointer;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+} */
```
Comment (Alusp) [inline style.css]: - [ ] You have done great job :clap: However, Please do not commit old pieces of code as inline comments. Leaving old pieces of code as comments can clutter the codebase and make it harder to read and understand. Over time, as more code comments accumulate, it becomes challenging to differentiate between actual code and commented-out sections. This clutter can hinder the development process and introduce confusion for developers trying to navigate and maintain the code. :slightly_smiling_face:

File: style.css [hunk 3]
```diff
@@ -567,10 +719,52 @@ textarea {
...
+    top: 20px;
+    box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+    width: 100%;
+    background: #fff;
+    margin: 16px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    transform: translate(-73%); */
+
+  /* } */
```
Comment (Alusp) [inline style.css]: - [ ] You have done great job :clap: However, Please do not commit old pieces of code as inline comments. Leaving old pieces of code as comments can clutter the codebase and make it harder to read and understand. Over time, as more code comments accumulate, it becomes challenging to differentiate between actual code and commented-out sections. This clutter can hinder the development process and introduce confusion for developers trying to navigate and maintain the code. :slightly_smiling_face:

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @JoyTaribagshaw, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- [ ] Good Commit History :green_circle: 
- [ ] Good Github Flow :green_circle: 
- [ ] Good PR Summary :green_circle: 


## Required Changes ♻️
_Check the comments under the review._

- [ ] Good job so far, :clap: Please ensure that you use a meaningful branch name next time, By adopting meaningful branch names, you improve collaboration, documentation, organization, and communication within your development team. It enhances the overall workflow, simplifies branch management, and promotes a better understanding of the codebase's history and current state. :slightly_smiling_face:   

- [ ] Good job implementing the changes and improving the UI of the project :clap:  however, there are still some adjustments to be made. Currently, your popup does not align with the design.  Kindly make some adjustments to your styling to implement these changes. Please check the image attached below for comparison and better understanding. :slightly_smiling_face:  

Below is an illustration of the difference: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

Student Desktop Design

</td>
      <td>

Figma Desktop

</td>
    </tr>
  </table> 


<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

Student Mobile Design

</td>
      <td>

Figma Mobile
</td>
    </tr>
  </table> 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (JoyTaribagshaw) [reply index.js]: Hy @Alusp ,please some issues here are not clear to me.

File: style.css (hunk 3 above)
Comment (JoyTaribagshaw) [reply style.css]: I have resolved this issue, thank you.

**Comment by JoyTaribagshaw** [comment]
Thank you for all your feedbacks @Alusp , I have made all the changes as seen in the images below.

# Mobile Screenshot
mobile-popup

# Desktop Screenshot
desktop-popup

**Review by topeogunleye — APPROVED** [review]
### Hi @JoyTaribagshaw, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Removed commented-out lines of code. ✔️ 
- Followed the Figma design in implementing UI ✔️ 
 
## [Optional] suggestions:

- Hi, thank you for your pull request. I noticed that you named your feature branch `develop`. This does not follow the convention of naming a branch after an action or a feature that you are working on. For example, if you are adding a login feature, you could name your branch added-login or login-feature. This makes it easier to identify the purpose of the branch and track the changes. For future pull requests, please use more descriptive and meaningful names for your feature branches. You can read more about the best practices for naming branches here: https://deepsource.io/blog/git-branch-naming-conventions/
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**